.github/workflows/build_navitia_packages_for_dev_multi_distribution.yml

```
@@ -51,3 +51,6 @@ jobs:
       run: apt update && apt install -y httpie
     - name: run artemis NG on https://jenkins-core.canaltp.fr/job/artemis_ng
       run: http -v -f POST https://${{secrets.JENKINS_NG_TOKEN}}@jenkins-core.canaltp.fr/job/artemis_ng/buildWithParameters
+    - name: run deploy on artemis machine https://jenkins-core.canaltp.fr/job/deploy-navitia
+      if: ${{ github.event_name == 'push' }}
```

are the `}}` at the right position ?
```suggestion
      if: ${{ github.event_name }} == 'push' 
```

---

i found this syntax here:  https://docs.github.com/pt/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsif
```
steps:
 - name: My first step
   if: ${{ github.event_name == 'pull_request' && github.event.action == 'unassigned' }}
   run: echo This event is a pull request that had an assignee removed.
```

---

allright, that's one of the mysteries of syntax !